docs/how-to/h-integrate-alerts.md

```diff
@@ -0,0 +1,206 @@
+# Integrate customer alerting rules and dashboards
+
+This guide shows you how to integrate an existing set of rules to your Charmed Kafka and Charmed Zookeeper deployment to be consumed with the [Canonical Observability Stack (COS)](https://charmhub.io/topics/canonical-observability-stack).
+To do so, we will sync the alert rules stored in a git repo to COS Lite.
+
+## Prerequisites
+
+Refer to the [Enable Monitoring](/t/charmed-kafka-documentation-how-to-enable-monitoring/10283) guide to deploy the cos-lite bundle in a Kubernetes environment and relate the Charmed Kafka model to the COS offers.
+The rest of this guide works on the assumption that two models are deployed: `cos` (on k8s) containing the cos-lite bundle and `kafka-dev` (on VM **TODO: adapt for k8s version**) containing 5 Zookeeper units and **2** Kafka.
```

Comment:
There should not be any difference between VM and k8s for the rest of the guide. The `grafana-agent` subordinate charm is mentioned in the Enable Monitoring guide